src/js/components/List/__tests__/List-test.js

```diff
@@ -1,13 +1,153 @@
 import React from 'react';
 import 'jest-styled-components';
-import { cleanup, render, fireEvent } from '@testing-library/react';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
 
+import { cleanup, render, fireEvent } from '@testing-library/react';
+import { axe } from 'jest-axe';
 import { Grommet } from '../../Grommet';
 import { List } from '..';
 
 describe('List', () => {
   afterEach(cleanup);
 
+  test('list should have no violations', async () => {
+    const { container } = render(
+      <Grommet>
+        <List />
+      </Grommet>,
+    );
+
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+
+  test('mouse events', () => {
+    const onClickItem = jest.fn();
```

Comment:
Not sure if we need the onClickItem if theres no click here in this test. Look at this throughout

Reply:
Without the onClickItem the mouseOut, MouseOver, focus, and blur events don't work, even though in this test the onClickItem isn't actually being called.

Reply:
Right. Focus and blur will occur on input component, and without the onClick this entry will be just a label without focus and blur capabilities.